src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_VALUE = 10;
+    private static final double INCREASE_INDEX = 1.5;
+    private int size;
+    private T[] array;
```

Comment:
naming after data types isn't very informative, rename to data, elements, or smth like that